### DISCUSSION TITLE
[Bug]: FixedLayout Group Prevents Selection of Elements Outside Its Bounds.

### DISCUSSION THREAD

**Discussion by xmejsieoke** [fabricjs/fabric.js#10290] [Q&A] [answered]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

6.0.2

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

None

### Link To Reproduction

https://jsfiddle.net/emd71hjo/7/

### Steps To Reproduce

1. Initialize a group positioned at [0, 0] with dimensions [300, 300]
2. Set the group's layout type to FixedLayout
3. Create a rectangle positioned at [300, 300] with arbitrary width and height
4. Add this rectangle to the group

### Expected Behavior

The rectangle, despite being outside the group's bounds, should remain selectable on the canvas.

### Actual Behavior

The rectangle cannot be selected on the canvas, suggesting the group limits interaction to elements within its defined [300, 300] boundaries.

**Comment by asturur** [comment]
Your expected behaviour is wrong. A fixed group is limited to its size, and not the inner object is not selectable outside those bounds.

I don't see how you would select something that is invisible sitting on top maybe of a visible object, it don't see that as a great default behavior.
You can try to build it in your app using the onDeselect callback and the pointer information.

**Reply by xmejsieoke** [reply, marked as answer]
In findTarget, replace this._objects with all layers on the canvas (recursive search is required). This way, the rectangle in this scenario can be selected.

**Reply by abhee235** [reply]
Hello @asturur , Yes there could be a case, eg. If we are extending the group class to create a frame like component, and we require to create a label for that frame. Frame Label would be outside for Frame(group), and it should except events, ( editable, events etc).

**Reply by asturur** [reply]
That is how you want to build your case, you can also build your own layout manager or targeting logic.
An alternative is that the label for the frame is not inside the frame but is grouped with the frame in a normal interactive group
